door43_tools/templaters.py

```diff
@@ -33,6 +38,9 @@ def run(self):
         self.rc = RC(self.source_dir)
         with open(self.template_file) as template_file:
             self.template_html = template_file.read()
+            soup = BeautifulSoup(self.template_html, 'html.parser')
+            soup.body['class'] = self.resource_type
```

Comment:
There is no class attribute in the template, but guess I could add to it just to be safe. Will research how to do it. 

And I thought it used to be two sets of div tags with the id of "content" that was occurring but fixed that a while ago. Where have you seen two sets of body tags?